packages/cms/src/elements/schemas/shared/index.ts

```diff
@@ -0,0 +1,72 @@
+import { snakeCase } from 'change-case';
+import { isDefined } from 'ts-is-present';
+
+export const scopes = [
+  { title: 'Actueel', value: 'ac' },
+  { title: 'Landelijk', value: 'nl' },
+  { title: 'Veiligheidsregio', value: 'vr' },
+  { title: 'Gemeente', value: 'gm' },
+  { title: 'Europe', value: 'eu' },
```

Comment:
do you mean international?

```suggestion
  { title: 'Internationaal', value: 'in' },
```

If not, then go for dutch:
```suggestion
  { title: 'Europa', value: 'eu' },
```